pkgs/top-level/all-packages.nix

```diff
@@ -22520,13 +22520,13 @@ with pkgs;
     inherit (darwin.apple_sdk_11_0.frameworks) OpenGL;
     inherit (darwin.apple_sdk_11_0.libs) Xplugin;
   };
-  mesa_23_0_1 = darwin.apple_sdk_11_0.callPackage ../development/libraries/mesa/23.0.1.nix {
+  mesa_23_0_2 = darwin.apple_sdk_11_0.callPackage ../development/libraries/mesa/23.0.2.nix {
```

Comment:
I'd personally keep two or so versions for each major version (_22, _23) in case there are regressions but it's not too important.

Reply:
We need to keep these two to avoid rebuild when this gets merged to `staging-next` where 23.0.1 is the default right now.